rskj-core/src/test/java/co/rsk/peg/BridgeTest.java

```diff
@@ -215,6 +203,54 @@ void increaseLockingCap_invalidParameter() {
         assertThrows(VMException.class, () -> bridge.execute(aboveMaxLengthData));
```

Comment:
Shouldn't we assert the specific exception type actually? BridgeIllegalArgument or LockingCapIllegalArgument

Reply:
I tried to do it when I refactored, but I got an issue since it throws VMException internally instead of the specific exception. Please take a look at the capture:
<img width="1088" alt="image" src="https://github.com/user-attachments/assets/457f3ceb-68bf-4b3d-97e5-7a328a91bde5">